intellij/dependencies.json

```
@@ -0,0 +1,26 @@
+{
+  "[2025.1,)": [
+    {
+      "tool": "java",
+      "versionRange": "[21,)"
+    }
+  ],
+  "[2024.1,2025.1)": [
+    {
+      "tool": "java",
+      "versionRange": "[21,)"
+    }
+  ],
+  "[2022.2,2024.2)": [
```

Make version ranges consistent and avoid overlaps.
```suggestion
  "[2022.2,2024.1)": [
```